debug/debug.go

```diff
@@ -1,386 +0,0 @@
-// Package debug for debug helper functions.
```

Comment:
@aaron-kasten could you please put just the file rename change as a separate PR before this PR ?

It is hard to review this PR unless we are able to look at the diffs. Due to the file renames the diffs are hard to see.

Reply:
You can separate these two things:

1. File renames in a separate first PR
2. File changes in a follow-up PR after that.

Reply:
I can do that.